tests/lib/nrf_modem_lib/nrf_modem_lib_trace/testcase.yaml

```diff
@@ -0,0 +1,6 @@
+tests:
+  nrf_modem_lib.nrf_modem_lib_trace:
```

Comment:
nit: `nrf_modem_lib.trace` ?

Reply:
As far as I've seen it's parent_directory.test_directory